src/app/templates/app/archive_requests.html

```diff
@@ -46,6 +46,9 @@
 
 <style>
     th {text-align: center;}
+    .clickableRow {cursor: pointer;}
+    table#requests_table.dataTable td:hover {background-color: #ddd;}
```

Comment:
We would like to show the whole row highlighted not a single data field. Use `tr` instead of `td`. Also, don't use the table id to set the style. This would be redundant if we change the name of the table in future. Rather use the style tag directly in the table tag above or set this default in the base.html `style` tag.

Reply:
Moved `<style>` to base.html, used css for `tr` tag without id.